controllers/healthcheck_controller.go

```diff
@@ -24,11 +24,14 @@ import (
 	"github.com/ghodss/yaml"
 	"github.com/go-logr/logr"
 	"github.com/prometheus/client_golang/prometheus"
+	v1 "k8s.io/api/core/v1"
```

Comment:
nit: may not need to use `v1` alias in this case since `v1` is already last item in path
```suggestion
	"k8s.io/api/core/v1"
```